css-text-3/line-breaking/reference/line-breaking-ic-001-ref.html

```diff
@@ -0,0 +1,23 @@
+<!doctype html>
+<html>
+<meta charset="utf-8">
+<title>Reference File for line breaking tests using ideographic caracters</title>
+<link rel=author title="Florian Rivoal" href="http://florian.rivoal.net">
+<!--
+	This is reference is theoretically identical to line-breaking-001.html,
+	but is needed nonetheless due to difference in anti-aliasing / font rendering.
```

Comment:
I can get rid of the comment if it bothers you, but it ought to be true because:
* both tests have `font-size: 20px` and `line-height: 1em`, so both have a line-height of 20px
* the `口` character is a CJK ideograph, and therefore its advance measure can be expected to be 1em in 99% of the fonts. It could be different, but in practice it will not be.
* If you combine the two points above, in line-breaking-ic-001-ref.html, each `口` is exactly a 20px square.
* The ink of of `口` is of course smaller than that, but it does not matter since the background color is set as well.
* In the X letter in ahem is defined to be a square exactly 1em in height and width.
* Both files then vertically stack two of these boxes.

=> Both files should result in a 20px × 40px vertical green rectangle.

In the case of line-breaking-ic-001-ref.html, this is what you get. In the case of line-breaking-001-ref.html, on standard resolution screens, you get that as well, and on high-res screen, you get approximately that modulo some fuzziness in the rendering of Ahem, probably for the same reasons we're having issues with the writing-modes test suite.

All the tests that use either reference use the same font (Ahem, or default CJK + background) as the reference they are pointing to, so we should not have the issues that the writing modes test suite is having. But if this Ahem fuzzyness issue was solved, we would be able to have a single ref instead of two.